pkg/k8s/pod.go

```diff
@@ -483,26 +522,33 @@ func (c *Client) buildInitContainerCommand(volumes []*Volume, files []*File) []s
 			cmds = append(cmds, parentDirCmd)
 			dirsProcessed[folder] = true
 		}
-		copyFileToKnuu := fmt.Sprintf("cp %s %s && ", file.Dest, filepath.Join(knuuPath, file.Dest))
-		cmds = append(cmds, copyFileToKnuu)
+		chown := file.Chown
+		permission := file.Permission
+		addFileToKnuu := fmt.Sprintf("cp %s %s && ", file.Dest, filepath.Join(knuuPath, file.Dest))
+		if chown != "" {
```

Comment:
why not use  the `file.Chown` itself, do we change it somewhere? same for permission

Reply:
What do you mean?
When mounting a file through a configMap, it does not preserve file owner and permissions from the original file. A ConfigMap has no knowledge about ownership or permissions. This is why we need to mount the ConfigMap in the init container, change the owner and permission, and then move it to a volume that is mounted to the main container.

Reply:
I meant why we assign it to a new variable. we could have it like so:

```go
if file.Chown != "" {
   ...
```